rust/Makefile

```diff
@@ -77,7 +82,7 @@ bindgen_c_flags = $(filter-out $(bindgen_skip_c_flags), $(c_flags)) \
 	$(bindgen_extra_c_flags)
 endif
 
-bindgen_opaque_types := xregs_state desc_struct arch_lbr_state
+bindgen_opaque_types := xregs_state desc_struct arch_lbr_state local_apic
```

Comment:
Is this one related or is another cleanup?

Reply:
This is in a separate commit. I couldn't build on my machine without this change, but it is trivial enough that I think it might not worth a separate PR.